apps/www/_blog/2023-01-05-supabase-beta-december-2022.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Supabase Beta December 2022
+description: This month the Beta Update is a Launch Week 6 Special, where we review the cascade of announcements.
+author: ant_wilson
+image: 2023-01-05-december-beta-update/monthly-update-december2022.jpg
+thumb: 2023-01-05-december-beta-update/monthly-update-december2022.jpg
+tags:
+  - release-notes
+date: '2023-01-05'
+toc_depth: 3
+---
+
+Happy New Year! Hope you are off to a great start. We closed 2022 on a high note with [Launch Week 6](https://supabase.com/launch-week) and a cascade of announcements. Here is everything we shipped so you can try them all 💻
+
+## Day 1: New Supabase Docs
```

Comment:
Should we take "Supabase" out of the titles? Seems redundant

Reply:
Makes sense from this one and Storage.